cla_public/templates/base.html

```diff
@@ -51,32 +54,46 @@
 
 {% block head %}
   <meta http-equiv="content-language" content="{{ request.cookies.get('locale', 'en')[:2] }}">
+{%- endblock %}
+
+{% block javascripts %}
+  <script src="{{ asset(filename='javascripts/cla.js') }}"></script>
+  {% if config.DEBUG %}
+    <script src="{{ url_for('static', filename='javascripts/cla-debug.js') }}"></script>
+  {% endif %}
+
+  <!-- Start of google analytics -->
   <script>
     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
     })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
 
-    ga('create', '{{ config.GA_ID }}', 'auto');
-    ga('set', 'anonymizeIp', true);
+    if (window.GOVUK.checkConsentCookieCategory('', 'usage')) {
+      ga('create', '{{ config.GA_ID }}', 'auto');
+      ga('set', 'anonymizeIp', true);
+    }
+
+    if (window.GOVUK.checkConsentCookieCategory('', 'brexit')) {
+      // GDS CROSS DOMAIN ANALYTICS
+      ga('create', 'UA-145652997-1', 'auto', 'govuk_shared', {'allowLinker': true});
```

Comment:
Not ideal for this to be hard-coded but I guess it's not the end of the world at all